doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -0,0 +1,136 @@
+.. _partition_mode:
+
+Using partition mode on UP2
+===========================
+This tutorial describes all the steps you need to take in order to use partition mode with the ACRN hypervisor on UP2.
+
+Enabling partition mode
+***********************
+1. Make the following changes in ``hypervisor/arch/x86/Kconfig`` :
+
+   .. code-block:: none
+
+      diff --git a/hypervisor/arch/x86/Kconfig b/hypervisor/arch/x86/Kconfig
+      index d4b41165..c279cd7f 100644
+      --- a/hypervisor/arch/x86/Kconfig
+      +++ b/hypervisor/arch/x86/Kconfig
+      @@ -13,7 +13,7 @@ endchoice
+
+      choice
+          prompt "Hypervisor mode"
+      -       default SHARING_MODE
+      +       default PARTITION_MODE
+
+      config SHARING_MODE
+          bool "Sharing mode"
+      @@ -72,7 +72,7 @@ endchoice
+      config SERIAL_MMIO_BASE
+          hex "Base address of serial MMIO region"
+          depends on SERIAL_MMIO
+      -       default 0xfc000000
+      +       default <Your serial mmio addr>
+
+
+2. Modify the ``vpci_vdev_arrary`` structure (one per vm) in ``hypervisor/partition/vm_description.c`` per 
```

Comment:
Changing the ACRN source code doesn't seem right either?  What happens when a new release of ACRN is made?  Is there a board or platform configuration parameter when building ACRN?  Having the user change ACRN source code seems unsupportable?

Reply:
I believe there is a series of patches coming about built-in VM configurations: https://lists.projectacrn.org/g/acrn-dev/message/13274. But until then, modifying the source code is the only option.

Can we all remember to modify this tutorial once the patch lands in the code?

Reply:
At second thought, I think it's not related as this one uses `acrn-dm` which is **not** used in the partition mode. Is there a similar mechanism coming or is editing the source code the right option here?

Reply:
Directly changing the ACRN source code seems weird to me as well. But for current implementation, this is the only way we configure the assigned devices info and boot parameters.

We can make those devices info configurable through menuconfig. But there will be a lot of  effort to do so.

@gvancuts @dbkinder @jren1 @ailun258. What's your opinions?